components/runtimes/python312/README_template.md

```diff
@@ -0,0 +1,45 @@
+# Ejected Functions Workspace
+
+This folder contains all most important block to test, run, deploy and productize your buisness application and consists of following elements:
+
+* `handler.py` - source code of the buisness application
+* `server.py` - server source code needed to run the handler.py
+* `requirements.txt` - dependencies file required for the handler.py and the server.py to run
+* `lib/` - directory containing server SDK (like cloudevents or tracing functionality)
+* `resources/` - directory with basic Kubernetes resources needed to deploy application on a cluster
+* `Dockerfile` - application image definition
+* `Makefile` - basic portion of automations and scripts
+   
+## Scripts and Automations
+
+The `Makefile` file is designed to speed up some processes like running application locally, build or deploy it on a k3d cluster.
+
+Read more about possibilities and functionalites by running the `make help` target.
+
+### To run application locally
+
+>NOTE: because application is run outside of the cluster it cannot simply reach in-cluster services and use container envs. It is strongly recommend to use following target to test application without such dependecies or mock them and export container envs manually.
```

Comment:
```suggestion
> [!NOTE] 
> Because the application is run outside the cluster, it cannot simply reach in-cluster services and use container envs. It is strongly recommended to use the following target to test the application without such dependencies or mock them and export container envs manually.
```